net-p2p/retroshare/retroshare-0.6.5.ebuild

```diff
@@ -0,0 +1,141 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/RetroShare/RetroShare.git"
+EGIT_COMMIT="v${PV}"
+EGIT_SUBMODULES=()
+
+inherit desktop eapi7-ver git-r3 gnome2-utils qmake-utils versionator
```

Comment:
- using both `eapi7-ver` and `versionator` is odd, surely a mistake?
- switch away from `gnome2-utils` and use `xdg_icon_cache_update` from `xdg-utils` instead.

Reply:
should be fixed now

Reply:
thanks!